pythia/common/task_loader.py

```diff
@@ -39,18 +39,20 @@ def get_test_reporter(self, dataset_type):
     def _load_task_config(self, task_name):
         directory = os.path.dirname(os.path.abspath(__file__))
         config_path = os.path.join(directory, "..", "tasks", task_name, "config.yml")
-        task_config = {}
+
+        self.task_config = {}
```

Comment:
Maybe use a property for `task_config` as a wrapper over a private variable of `_task_config`